src/main/java/ApplyToGroup/ApplyToGroupController.java

```diff
@@ -0,0 +1,18 @@
+package ApplyToGroup;
+
+public class ApplyToGroupController {
+
+    final ApplyToGroupInputBoundary applyToGroupInput;
+
+    public ApplyToGroupController(ApplyToGroupInputBoundary applyGateway){
+        this.applyToGroupInput = applyGateway;
+    }
+
+    ApplyToGroupResponseModel create(String groupName) {
+        ApplyToGroupRequestModel requestModel = new ApplyToGroupRequestModel(groupName);
+
+        return applyToGroupInput.create(requestModel);
+    }
+
```

Comment:
I believe this should be in the package with your screens to abide by clean architecture because the controller is not in the use case layer

Reply:
Yes, thank you aarya, i had not worked on the screens package just yet.